lib/shopify_cli/services/app/create/rails_service.rb

```diff
@@ -109,7 +109,9 @@ def check_dependencies
 
           def check_ruby
             ruby_version = Environment.ruby_version(context: context)
-            return if ruby_version.satisfies?("~>2.5") || ruby_version.satisfies?("~>3.1.0")
+            return if ruby_version.satisfies?("~>2.5") ||
+              ruby_version.satisfies?("~>3.0.0") ||
```

Comment:
Noting this change that snuck in: Apparently we were not allowing Ruby 3.0.x which is also a big problem!